packages/lambda-tiler/src/routes/import.ts

```diff
@@ -43,16 +43,18 @@ export async function Import(req: LambdaHttpRequest): Promise<LambdaHttpResponse
     ctx.override!.id = id;
     ctx.outputLocation.path = fsa.join(ctx.outputLocation.path, id);
 
-    // Start processing job
-    await CogJobFactory.create(ctx);
+    // Insert Processing job config
     jobConfig = {
       id: jobId,
       name: path,
       status: 'processing',
     } as ConfigProcessingJob;
 
-    const config = new ConfigProviderDynamo(Const.TileMetadata.TableName);
-    await config.ProcessingJob.put(jobConfig);
+    if (Config.ProcessingJob instanceof ConfigDynamoBase) await Config.ProcessingJob.put(jobConfig);
```

Comment:
I made https://github.com/linz/basemaps/pull/2183 to make this easier in the future.

Reply:
Thanks. Just Updated. Can you plz give another go.